### PR TITLE
Convert to project template

### PR DESCRIPTION
**Pull request by pmdarrow**
- Convert django-base to Django project template.
- Create `new_project.sh` to start a new project
- Update `setup.sh` to get developers bootstrapped on an existing project
- Make `setup.sh` runnable more than once

### CODE REVIEW

File: new_project.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Only use this script if you are bootstrapping a new project. Once a project
```
Comment (wolfe) [inline new_project.sh:3]: Guard with check for whether the project is new?  Just check of project_name is a string in an appropriate file; whichever is changed first.

**Comment by wolfe** [comment]
I think we already discussed all needed.  Looks good otherwise.

File: new_project.sh (hunk 1 above)
Comment (pmdarrow) [inline new_project.sh:3]: This file is removed at the end of the script so theres no risk of this being run on an existing project.

Comment (wolfe) [inline new_project.sh:3]: OK, but that assumes the script runs properly to completion.

**Pull request merged**